llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -1462,6 +1462,7 @@ void HWAddressSanitizer::instrumentStack(memtag::StackInfo &SInfo,
     Value *AICast = IRB.CreatePointerCast(AI, PtrTy);
 
     auto HandleLifetime = [&](IntrinsicInst *II) {
+      // FIXME: Remove this code entirely.
```

Comment:
I just submitted https://github.com/llvm/llvm-project/pull/150510, so we can just remove the whole thing now.